### PR TITLE
Fix warnings in exiv2

### PR DESCRIPTION
**Pull request by qarkai**
Also fix indentation.

### CODE REVIEW

File: src/exiv2.cc [hunk 1]
```diff
@@ -648,7 +649,7 @@ char *exif_item_get_data(ExifItem *item, guint *data_len)
 		auto md = reinterpret_cast<Exiv2::Metadatum *>(item);
 		if (data_len) *data_len = md->size();
 		auto data = static_cast<char *>(g_malloc(md->size()));
-		long res = md->copy(reinterpret_cast<Exiv2::byte *>(data), Exiv2::littleEndian /* should not matter */);
+		size_t res = md->copy(reinterpret_cast<Exiv2::byte *>(data), Exiv2::littleEndian /* should not matter */);
```
Comment (neheb) [inline src/exiv2.cc:652]: auto?

Comment (qarkai) [reply src/exiv2.cc]: ok

**Comment by caclark** [comment]
When running `./scripts/test-all.sh` I get this error:

`test:         Code Correctness_ exiv2.cc>
start time:   19:30:04
duration:     14.86s
result:       exit status 1
command:      MALLOC_PERTURB_=108 /usr/bin/clang-tidy -p ./build -quiet --config-file /home/cclark/data/geeqie/geeqie/.clang-tidy /home/cclark/data/geeqie/geeqie/build/../src/exiv2.cc
----------------------------------- stdout -----------------------------------
../src/exiv2.cc:959:24: error: comparison of integers of different signs: 'size_t' (aka 'unsigned long') and 'long' [clang-diagnostic-sign-compare,-warnings-as-errors]
                for (size_t i = 0; i < item.count(); i++)
                                   ~ ^ ~~~~~~~~~~~~
`

Clearly I still have not fixed the GitHub action problem....

**Comment by qarkai** [comment]
> When running `./scripts/test-all.sh` I get this error:
> 
> `test: Code Correctness_ exiv2.cc> start time: 19:30:04 duration: 14.86s result: exit status 1 command: MALLOC_PERTURB_=108 /usr/bin/clang-tidy -p ./build -quiet --config-file /home/cclark/data/geeqie/geeqie/.clang-tidy /home/cclark/data/geeqie/geeqie/build/../src/exiv2.cc ----------------------------------- stdout ----------------------------------- ../src/exiv2.cc:959:24: error: comparison of integers of different signs: 'size_t' (aka 'unsigned long') and 'long' [clang-diagnostic-sign-compare,-warnings-as-errors] for (size_t i = 0; i < item.count(); i++) ~ ^ ~~~~~~~~~~~~ `

Exiv2 have changed types in v0.28. Fixed.

> Clearly I still have not fixed the GitHub action problem....

Have you considered using dedicated clang-tidy action? E.g. one of https://github.com/marketplace?category=&type=&verification=&query=clang-tidy.

**Comment by caclark** [comment]
Implemented in commit b71833996d3aecf3efdf25

**Pull request closed without merging**